P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -2664,6 +2664,21 @@ referring to the zone marked in purple on the right
             annotation. </p>
       </div>
    </div>
+   
+   
+   <div  type="div2" xml:id="PHRB">
+      <head>Transcription and Ruby</head>
+      
+      <p>These Guidelines also provide special elements to support the encoding of
+         ruby annotations, which are common in East Asian textual traditions. These
+         elements provide yet another method of capturing a specific type of annotation, 
```

Comment:
This sentence “These elements provide … text and annotations.” is a bit of a run-on. It should at least be punctuated with semicolons to help the reader see the major clauses, but better yet divvied into 2 or 3 sentences.

Reply:
This is a short paragraph already divided into three sentences, of which the longest is has only 39 words. AFAICS there's nothing syntactically wrong with the longest; it's not a run-on sentence (https://www.grammarly.com/blog/run-on-sentence-basics/). I think if you believe it should be changed, you need to provide your suggested version.

Reply:
First crack:
~~~xml
<p>These Guidelines also provide special elements to support the
encoding of ruby annotations, which are common in East Asian textual
traditions. These elements provide a method of capturing a specific
type of annotation, in addition to the generic methods like the
<gi>note</gi> or <gi>interp</gi> elements. Both the specific and
general methods should integrate well with the transcriptional
elements described above, allowing authorial and scribal features to
be captured in conjunction with ruby base text and annotations. See
<ptr target="#COHTGRB"/> for more information about these
elements.</p>
~~~

Reply:
Done! Thanks.